io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/ReaderSettings.java

```diff
@@ -201,7 +201,8 @@ else if (skipType) {
 		}
 
 		// read dateFormat
-		String dateFormatString = reader.getParameter(PARAMETER_DATE_FORMAT).as(String.class);
+		String dateFormatString = reader.getParameter(CommonSchemaConstants.PARAM_DATE_FORMATTER)
```

Comment:
Shouldn't this rather set [this parameter](https://github.com/emanuelaepure10/hale/blob/8deb946f73d1727e46ab7eda35835ec92866b4e5/io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/ReaderSettings.java#L60) that was introduced as part of [this PR](https://github.com/halestudio/hale/pull/1091)?

Reply:
YES! Thanks